frontend/javascripts/oxalis/view/help_modal.tsx

```diff
@@ -0,0 +1,98 @@
+import { sendHelpEmail, updateNovelUserExperienceInfos } from "admin/admin_rest_api";
+import { Modal, Input, Alert } from "antd";
+import { enforceActiveUser } from "oxalis/model/accessors/user_accessor";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import { OxalisState } from "oxalis/store";
+import React, { CSSProperties, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+function HelpButton() {
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  const dispatch = useDispatch();
+  const activeUser = useSelector((state: OxalisState) => enforceActiveUser(state.activeUser));
```

Comment:
this crashes wk when the user is not logged in. why use `enforceActiveUser` when you are guarding against `!activeUser` later, anyway?

Reply:
Copy pasta. That's why 😊

Reply:
Fixed